smoothie.js

```diff
@@ -671,6 +671,11 @@
       var timeSeries = this.seriesSet[d].timeSeries,
           dataSet = timeSeries.data,
           seriesOptions = this.seriesSet[d].options;
+          
+      if (seriesOptions.lineWidth === 0) {
+        context.restore();
+        continue;
```

Comment:
Does this also prohibit using a line width of zero and a fill style in order to have a filled curve with no line?